.drone.yml

```
@@ -94,7 +94,11 @@ steps:
     GCLOUD_AUTH:
       from_secret: gcloud_auth
   commands:
-  - dapper -f Dockerfile.test.dapper
+  - docker build --target test-k3s -t k3s:test-${DRONE_STAGE_ARCH} -f Dockerfile.test .
```

The tags also need to be unique I think, otherwise you can only have one job a a time on the host before the tags will start getting overwritten.
```suggestion
  - docker build --target test-k3s -t k3s:test-${DRONE_STAGE_ARCH}-${DRONE_COMMIT} -f Dockerfile.test .
```

---

Done!